t/plugin/server-info.t

```diff
@@ -133,3 +133,54 @@ integral
 [error]
 --- error_log
 timer created to report server info, interval: 60
+
+
+
+=== TEST 3: get server_info from plugin control API
+--- yaml_config
+apisix:
+    id: 123456
+    enable_control: true
```

Comment:
This configuration may confuses beginner, because the control server in the test is hardcoded in APISIX.pm, instead of generating from config.yaml.

Reply:
Fixed.